src/devices_models/devices/storage.jl

```diff
@@ -1,9 +1,16 @@
 #! format: off
 
 abstract type AbstractStorageFormulation <: AbstractDeviceFormulation end
+abstract type EnergyManagementFormulation <: AbstractStorageFormulation end
+abstract type EnergyTargetFormulation <: EnergyManagementFormulation end
+abstract type RelaxedEnergyTargetFormulation <: EnergyManagementFormulation end
 struct BookKeeping <: AbstractStorageFormulation end
 struct BookKeepingwReservation <: AbstractStorageFormulation end
-
+struct EndOfPeriodStaticEnergyTarget <: EnergyTargetFormulation end
```

Comment:
We need docstrings for these formulations. It isn't at all clear what the model is from the description.